### PR TITLE
[Localization] Help Text for Command Switches

### PR DESCRIPTION
**Pull request by JavierMatosD**
Localizes the help text for the following commands and associated options:
* add-version
* ci
* civerifyversions
* contact
* depend-info
* edit
* env
* fetch
* find
* generate-message-map
* info
* list
* new
* regenerate
* search
* set-installed
* update-baseline
* upgrade
* x-download
* install
* export
* remove

Will sort messages after review :)

### CODE REVIEW

File: include/vcpkg/base/messages.h [hunk 1]
```diff
@@ -2095,5 +2095,221 @@ namespace vcpkg
                     (msg::path),
                     "",
                     "Failed to parse {path}, expected a top-level object.");
-
+    DECLARE_MESSAGE(HelpTxtOptDryRun, (), "", "Do not actually build or install.");
```
Comment (valeriaconde) [inline include/vcpkg/base/messages.h:2098]: nit but, is there a reason why some of these messages end with a period and others do not? should we be consistent?

File: include/vcpkg/base/messages.h [hunk 2]
```diff
@@ -2095,5 +2095,221 @@ namespace vcpkg
...
+    DECLARE_MESSAGE(CmdFormatManifestOptConvertControl, (), "", "Convert CONTROL files to manifest files.");
+    DECLARE_MESSAGE(CmdGenerateMessageMapOptOutputComments,
+                    (),
+                    "",
+                    "When generating the message map, include comments (the default)");
+    DECLARE_MESSAGE(
+        CmdGenerateMessageMapOptNoOutputComments,
+        (),
+        "",
+        "When generating the message map, exclude comments (useful for generating the english localization file)");
```
Comment (valeriaconde) [inline include/vcpkg/base/messages.h:2214]:
```suggestion
        "When generating the message map, exclude comments (useful for generating the English localization file)");
```

File: include/vcpkg/base/messages.h [hunk 3]
```diff
@@ -2095,5 +2095,221 @@ namespace vcpkg
...
+                    "",
+                    "proceeds with the (potentially dangerous) action without confirmation");
+    DECLARE_MESSAGE(CmdRegenerateOptDryRun,
+                    (),
+                    "",
+                    "does not actually perform the action, shows only what would be done");
+    DECLARE_MESSAGE(CmdRegenerateOptNormalize, (), "", "apply any deprecation fixups");
+    DECLARE_MESSAGE(HelpTextOptFullDesc, (), "", "Do not truncate long text");
+    DECLARE_MESSAGE(CmdSetInstalledOptDryRun, (), "", "Do not actually build or install");
+    DECLARE_MESSAGE(CmdSetInstalledOptNoUsage, (), "", "Don't print cmake usage information after install.");
```
Comment (valeriaconde) [inline include/vcpkg/base/messages.h:2238]:
```suggestion
    DECLARE_MESSAGE(CmdSetInstalledOptNoUsage, (), "", "Don't print CMake usage information after install.");
```

File: include/vcpkg/base/messages.h [hunk 4]
```diff
@@ -2095,5 +2095,221 @@ namespace vcpkg
...
+                    "Don't install the default features from the top-level manifest (manifest mode).");
+    DECLARE_MESSAGE(HelpTxtOptEnforcePortChecks,
+                    (),
+                    "",
+                    "Fail install if a port has detected problems or attempts to use a deprecated feature");
+    DECLARE_MESSAGE(HelpTxtOptAllowUnsupportedPort,
+                    (),
+                    "",
+                    "Instead of erroring on an unsupported port, continue with a warning.");
+    DECLARE_MESSAGE(HelpTxtOptNoUsage, (), "", "Don't print cmake usage information after install.");
```
Comment (valeriaconde) [inline include/vcpkg/base/messages.h:2132]:
```suggestion
    DECLARE_MESSAGE(HelpTxtOptNoUsage, (), "", "Don't print CMake usage information after install.");
```

File: locales/messages.json [hunk 5]
```diff
@@ -163,6 +179,82 @@
...
+  "CmdEditOptBuildTrees": "Open editor into the port-specific buildtree subfolder",
+  "CmdEnvOptions": "Add installed {path} to {env_var}",
+  "_CmdEnvOptions.comment": "An example of {path} is /foo/bar. An example of {env_var} is VCPKG_DEFAULT_TRIPLET.",
+  "CmdExportOpt7Zip": "Export to a 7zip (.7z) file",
+  "CmdExportOptChocolatey": "Export a Chocolatey package (experimental feature)",
+  "CmdExportOptDebug": "Enable prefab debug",
+  "CmdExportOptDryRun": "Do not actually export.",
+  "CmdExportOptIFW": "Export to an IFW-based installer",
+  "CmdExportOptInstalled": "Export all installed packages",
+  "CmdExportOptMaven": "Enable maven",
```
Comment (valeriaconde) [inline locales/messages.json:203]: nit but if this refers to Apache Maven I would capitalize all references

File: locales/messages.json [hunk 6]
```diff
@@ -163,6 +179,82 @@
...
+  "CmdExportSettingNugetVersion": "Specify the version for the exported NuGet package",
+  "CmdExportSettingOutput": "Specify the output name (used to construct filename)",
+  "CmdExportSettingOutputDir": "Specify the output directory for produced artifacts",
+  "CmdExportSettingPkgDir": "Specify the temporary directory path for the repacked packages",
+  "CmdExportSettingPrefabArtifactID": "Artifact Id is the name of the project according maven specifications",
+  "CmdExportSettingPrefabGroupID": "GroupId uniquely identifies your project according maven specifications",
+  "CmdExportSettingPrefabVersion": "Version is the name of the project according maven specifications",
+  "CmdExportSettingRepoDir": "Specify the directory path for the exported repository",
+  "CmdExportSettingRepoURL": "Specify the remote repository URL for the online installer",
+  "CmdExportSettingSDKMinVersion": "Android minimum supported sdk version",
```
Comment (valeriaconde) [inline locales/messages.json:223]:
```suggestion
  "CmdExportSettingSDKMinVersion": "Android minimum supported SDK version",
```

File: locales/messages.json [hunk 7]
```diff
@@ -163,6 +179,82 @@
...
+  "CmdExportSettingSDKTargetVersion": "Android target sdk version",
+  "CmdFetchOptXStderrStatus": "Direct status/downloading messages to stderr rather than stdout. (Errors/failures still go to stdout)",
+  "CmdFormatManifestOptAll": "Format all ports' manifest files.",
+  "CmdFormatManifestOptConvertControl": "Convert CONTROL files to manifest files.",
+  "CmdGenerateMessageMapOptNoOutputComments": "When generating the message map, exclude comments (useful for generating the english localization file)",
```
Comment (valeriaconde) [inline locales/messages.json:228]:
```suggestion
  "CmdGenerateMessageMapOptNoOutputComments": "When generating the message map, exclude comments (useful for generating the English localization file)",
```

File: include/vcpkg/base/messages.h (hunk 1 above)
Comment (JavierMatosD) [reply include/vcpkg/base/messages.h:2098]: I've been trying to add periods to actual sentences, but sometimes the messages are sentence fragments, and adding a period feels weird.

**Review by BillyONeal — APPROVED** [review]
LGTM! Please double check that you are happy with the one nitpick change I pushed.

**Pull request merged**